aviary/utils/process_input_decks.py

```diff
@@ -73,6 +100,11 @@ def parse_inputs(vehicle_deck, aircraft_values: AviaryValues(), meta_data=_MetaD
 
             # remove any elements that are empty (caused by trailing commas or extra commas)
             data_list = [dat for dat in data.split(',') if dat != '']
+
+            # continue if there's no data in the line but there are commas
```

Comment:
Since we can process these line-by-line, maybe we could add support for comments too (line startswith #)

Reply:
Lines 94-99 do that

Reply:
actually, we might be able to remove 98 and 99?

Reply:
we need 98-99 for when lines contain only commas. I updated the in-line comment